example/example.cpp

```diff
@@ -279,7 +279,7 @@ struct fmt::formatter<my_type> : fmt::formatter<std::string>
 {
     auto format(my_type my, format_context &ctx) -> decltype(ctx.out())
     {
-        return format_to(ctx.out(), "[my_type i={}]", my.i);
+        return fmt::format_to(ctx.out(), "[my_type i={}]", my.i);
```

Comment:
this is also an error that arises when compiling spdlog using the standard set to C++20. Very simple fix, just add fmt namespace to the front and the code can compile using C++20. If this PR doesn't get merged this should at least be fixed.